packages/keycloak-service/src/service/tests/index.test.ts

```diff
@@ -211,7 +211,7 @@ describe('services/keycloak', () => {
       'https://keycloak-test.smartregister.org/auth/realms/'
     );
     const result = await usersService.create(keycloakUser);
-    expect(result).toEqual({});
+    expect(result).toEqual(expect.any(Object));
```

Comment:
would it make for less brittle tests if this expected the data object mocked as the response to the fetch call, other than just any object

Reply:
I think i did this because the uuid in the payload updates every time we run tests.